partiql-ast/src/main/java/org/partiql/ast/v1/type/TypeAny.java

```diff
@@ -0,0 +1,24 @@
+package org.partiql.ast.v1.type;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class TypeAny extends Type {
```

Comment:
I'd defer to @jpschorr, but this may be a good opportunity for us to use the `TypeDynamic` naming since these changes are breaking anyways.